test/quantization/fx/test_quantize_fx.py

```diff
@@ -447,6 +448,114 @@ def test_linear_tanh_not_fused_by_default(self):
             expected_node_list=expected_nodes,
             expected_node_occurrence=expected_occurrence)
 
+    def test_fuse_conv_bn_add_relu_onednn(self):
+        # conv - bn - add - relu is fused for onednn backend only
+        from torch.ao.quantization.backend_config import get_onednn_backend_config
+        expected_nodes = [
+            ns.call_module(nni.ConvAdd2d),
+        ]
+        expected_occurrence = {
+            ns.call_module(nni.ConvAdd2d): 1,
+            ns.call_module(nn.BatchNorm2d): 0,
+        }
+
+        options = itertools.product(
+            [True, False],  # with_bn
+            [False],  # with_relu
```

Comment:
is relu supported?

Reply:
Oh, yes. The support of relu is added in the following up 2 PRs.